content/post/089-ipfs-mobile-design-research-findings.md

```diff
@@ -0,0 +1,60 @@
+
+![Status.im's key recovery process](https://raw.githubusercontent.com/ipfs/mobile-design-guidelines/master/.gitbook/assets/status-recover.png)
+
+After in-depth review, we found many intriguing approaches to several shared core areas:
+
+* Apps differed widely in how they approached onboarding and user education.
+* Users were generally made aware of privacy and connection implications with sharing.
+* Identity and account management varied, ranging from typical email login and password to key and seed phrase recovery.
+* We found status and notifications around connection, syncing, and download status to be lacking. We found many scenarios where the user could connect to others and send files without confirmation, as well.
+* In general, there was no consistent sharing method. The main sharing methods were either through torrent apps or through messaging, either direct to users or to community boards.
+* The anonymous file share web apps handled privacy better, making use of links and QR codes. 
+* To find other users with chat, features ranged from public discoverability to obscure invite methods.
+
+## User Research
+
+User research began with creating assumptions. We gathered these by speaking with and running a workshop with IPFS stakeholders. These assumptions framed the interviews with the Experts, Early Adopters, or Potential Users.
+
+#### EXPERTS
+
+In the interviews, the Experts expressed concern for many potential user issues such as battery life and signal. They did not have extensive user feedback to validate this yet and spoke about needing to grow their user bases. There was particular emphasis on identity being essential to both users and developers. There was also the assumption users concerned about privacy and security would find P2P useful. We also found that offline-first is a trend that is important but difficult to push forward. Design came up in interviews as well, pointing to Dropbox and AirDrop as the gold standard for file sharing. Finally, they highlighted status and notifications as areas requiring consideration and further work.
+
+#### EARLY ADOPTERS
+
+Early Adopters thought users should always know what was going on. The app would inform the user with notifications, but they should not be overwhelming. Early Adopters thought privacy and security was crucial. Its management, yet in general was difficult even for advanced users. They emphasised file management, security, permissions, and backup as essential. What was less important was who had the final or original file which was interesting. Most mentioned Single Sign-On (SSO) as a good thing but complex to manage. Finally, most mentioned cloud services like Dropbox as easy to use but not trustworthy.
```

Comment:
The "Its management” in the third sentence of Line 50 is not clear whether you are referencing privacy or security or both.

Reply:
Fixed, thanks!